docs/ota-client-guide/modules/ROOT/pages/meta-updater-provisioning-methods.adoc

```diff
@@ -0,0 +1,22 @@
+= Manual provisioning
+
+//MC: TOMERGE: Looks mostly like a duplicate of this topic: https://github.com/advancedtelematic/aktualizr/blob/master/docs/ota-client-guide/modules/ROOT/pages/simulate-device-cred-provtest.adoc
```

Comment:
Yeah, I think this one is entirely redundant.

Reply:
Ditto above